rancid-tomatillos/src/MoviePage/MoviePage.js

```diff
@@ -1,8 +1,42 @@
-import React from 'react'
+import React from "react";
+import api from '../api';
 
-function ShowPage(props) {
-    return <h1>isworking</h1>
-}
+function MoviePage(props) {
+  let id = props.location.pathname.split("/")[2];
 
+  if (props.match.isExact) {
+    console.log(props);
+    const {
+      average_rating,
+      backdrop_path,
+      budget,
+      genres,
+      id,
+      overview,
+      poster_path,
+      release_date,
+      revenue,
+      runtime,
+      tagline,
+      title,
```

Comment:
I tried a few things to short hand this with no success. I saw reference to Object.entries for iterating while destructuring, but couldn't figure it out. May be worth just long-handing inside the JSX since this is so bloated.